ensemble_eeg/ensemble_edf.py

```diff
@@ -399,11 +399,11 @@ def anonymize_edf_header(fd):
     startdate = datetime.strptime("01-01-1985", "%d-%m-%Y") + timedelta(age_in_days)
 
     # use MONTH DICT to bypass local language month abreviations
-    startdate_str = startdate.strftime('%d-%m-%Y')
-    month = re.search(r'-(0|1)\d-', startdate_str)[0]
+    startdate_str = startdate.strftime("%d-%m-%Y")
```

Comment:
At this point we might as well use:
```python
    startdate_str = f"{startdate.date.day:0>2}-{MONTH_DICT[startdate.date.month]}-{startdate.date.year:0>4})
```

You need to change the key type of `MONTH_DICT` from `str` to `int`.